modules/nw-installing-external-dns-operator-cli.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-external-dns-operator-cli_{context}"]
+= Installing the External DNS Operator by using the CLI
+
+You can install the External DNS Operator by using the CLI.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* You are logged into the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create an `OperatorGroup` object:
+
+.. Create a YAML file that defines the `OperatorGroup` object:
++
+.Example `operatorgroup.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: aws-lb-operatorgroup
+  namespace: external-dns-operator
+spec:
+  upgradeStrategy: Default
+----
+
+.. Create the `OperatorGroup` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f operatorgroup.yaml
+----
+
+. Create a `Subscription` object:
+
+.. Create a YAML file that defines the `Subscription` object:
++
+.Example `subscription.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: external-dns-operator
+  namespace: external-dns-operator
+spec:
+  channel: stable-v1
+  installPlanApproval: Automatic
+  name: external-dns-operator
+  source: qe-app-registry
```

Comment:
```suggestion
  source: redhat-operators
```

Btw I saw the same catalog source in ALBO. It should be the official Red Hat operators catalog source, not QE.